src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -73,6 +73,20 @@
  * </li>
  * </ul>
  * <p>
+ * To run the check with its default configuration:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * void method() {
+ *   int i = 5; // OK
+ *   System.out.println(i); // OK
```

Comment:
please add case with `// violation` for default, in other case it is not clear what code it violates in default mode.